src/test/regress/expected/multi_index_statements.out

```diff
@@ -357,13 +357,13 @@ SET client_min_messages TO DEBUG1;
 CREATE INDEX ix_test_index_creation2
     ON test_index_creation1 USING btree
     (tenant_id, timeperiod);
-DEBUG:  the index name on the shards of the partition is too long, switching to sequential and local execution mode to prevent self deadlocks: test_index_creation1_p2020_09_26_10311_tenant_id_timeperiod_idx
```

Comment:
I don't like the these changing debug messages.

We should normalize the outputs here. Should I do that here @onderkalaci ?